tests/e2e-pw/types/block-data.ts

```diff
@@ -1,5 +1,6 @@
 export type BlockData< T = unknown > = {
 	name: string;
+	slug: string;
 	mainClass: string;
-	selectors: Record< 'editor' | 'frontend', Record< string, unknown > >;
+	selectors: Record< 'editor' | 'frontend', Record< string, string > >;
```

Comment:
Not sure about this change. It could be necessary having an object like this:

```
	selectors: {
		frontend: {
			header: '.blabla',
			footer: '.blablabla',
		},
	},
```

Reply:
We should try to define this more robustly, I'll revert this for now!